qmi/instruments/newport/single_axis_motion_controller.py

```diff
@@ -138,6 +139,13 @@ def controller_address(self, controller_address: Optional[int]) -> None:
             raise QMI_InstrumentException(
                 f"Invalid controller address {self._controller_address}")
 
+    @contextmanager
```

Comment:
Can we unittest the context manager separately?

Reply:
Coverage reports 100% so I don't think there's a need for it. Perhaps if it would be a more complex one with inputs and/or logic, but now it is a very simple one so no.